### PR TITLE
Add the standard file globbing that grunt offers

### PR DESCRIPTION
**Pull request by MattCBowman**
I updated this plugin to make of use grunt's file globbing/expand/flatten/cwd capability.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     }
   ],
   "engines": {
-    "node": ">= 0.10.0"
+    "node": ">= 0.12.0"
```
Comment (robotlovesyou) [inline package.json]: I don't really think we should be updating the project to only run on >= 0.12.0 so quickly

**Comment by robotlovesyou** [comment]
This looks good but it needs to still run on node >= 0.10.0 because a lot of people haven't upgraded yet.

**Comment by MattCBowman** [comment]
Ok, I'm working on a fix for that now. The reason I pulled 0.12.0 was because of the addition of the file/directory path parsing and assembly. Would you be ok with importing the latest path library (0.12.0 stable) and using that in place of the version specific implementation?

**Comment by MattCBowman** [comment]
Hey @robotlovesyou - do you think this is something you'll be able to integrate? We've really liked using this library, but are currently pinned to my fork. We're moving on to the next thing and we'd prefer to point to your repo to take advantage of any future updates. Thanks!

**Comment by robotlovesyou** [comment]
Hello Matt, sorry very busy couple of days so didn't have a chance to look at this yet. The size of the import does make it feel a little 'hammer to crack a nut'. What exactly is it improving? I'm not saying no, I just want to be sure there isn't a more subtle way of achieving the same goal. Either way I need to run the latest patch against 0.10.x and I won't have time to do that until this evening (GMT)

**Comment by MattCBowman** [comment]
@robotlovesyou - I'll see if I have time to try and make it work with the 0.10.0 version of path this morning (EDT). Just so you know, I did test it out on 0.10.0 and 0.12.0.

**Comment by MattCBowman** [comment]
@robotlovesyou I've updated the PR to remove the reliance on the 0.12.0 path library.

**Pull request merged**